### ISSUE TITLE
OVA does not work if features column is not "Features"

### ISSUE THREAD

**Issue by xadupre** [dotnet/machinelearning#1259]
### System information

- **Windows**: 10.0.17134
- **.NET Version (eg., dotnet --info)**: 2.1.402

### Issue

OVA still looks for columns "Features" and cannot find it in the Following test. 

### Source code / logs

        [Fact]
        public void New_MetacomponentsFeaturesRenamed()
        {
            using (var env = new LocalEnvironment())
            {
                var data = new TextLoader(env, MakeIrisTextLoaderArgs())
                    .Read(new MultiFileSource(GetDataPath(TestDatasets.irisData.trainFilename)));

                var sdcaTrainer = new LinearClassificationTrainer(env, "Vars", "Label", advancedSettings: (s) => { s.MaxIterations = 100; s.Shuffle = true; s.NumThreads = 1; });
                var pipeline = new ConcatEstimator(env, "Vars", "SepalLength", "SepalWidth", "PetalLength", "PetalWidth")
                    .Append(new TermEstimator(env, "Label"), TransformerScope.TrainTest)
                    .Append(new Ova(env, sdcaTrainer))
                    .Append(new KeyToValueEstimator(env, "PredictedLabel"));

                var model = pipeline.Fit(data);
            }
        }
